### PR TITLE
add tokens to list in add account modal

### PR DESCRIPTION
**Pull request by valpinkman**
Possibility to see the ERC20 Tokens in the select component when adding a new account
Also added the page when selecting ERC20 when adding accont

### Type

Feat

### Context

LL-1846

### Parts of the app affected / Test plan

Add account modal

IMG_C7E113ACDC80-1
IMG_94470110A957-1

### CODE REVIEW

File: src/locales/en/common.json [hunk 1]
```diff
@@ -993,7 +993,12 @@
     "synchronizing": "Synchronizing",
     "synchronizingDesc": "We are synchronizing your accounts, this can take a while...",
     "noAccountToCreate": "No <1><0>{{currencyName}}</></> account was found to create. Restart the flow and sync your accounts",
-    "cantCreateAccount": "There are no transactions on your last created <1><0>{{accountName}}</></> account. You must first receive crypto assets on that account before you can add a new one."
+    "cantCreateAccount": "There are no transactions on your last created <1><0>{{accountName}}</></> account. You must first receive crypto assets on that account before you can add a new one.",
+    "tokens": {
+      "title": "Add Token",
+      "disclaimer": "{{tokenName}} is a ERC20 Token.\nYou can receive tokens directly on a Ethereum account",
```
Comment (juan-cortes) [inline src/locales/en/common.json]: _"**an** ERC20 Token" perhaps?
"**an** Ethereum account"_
@dasilvarosa

Comment (valpinkman) [reply src/locales/en/common.json]: I thought so too but I used the text from Zeplin

Comment (juan-cortes) [reply src/locales/en/common.json]: 🤷‍♂

**Comment by juan-cortes** [comment]
Wait for @dasilvarosa to merge for the wording check?

**Comment by valpinkman** [comment]
@juan-cortes yas

**Comment by gre** [comment]
@IAmMorrow NPRB means No Problem Really Beautiful?

**Comment by juan-cortes** [comment]
Google says _North Pacific Research Board_, what's going on @IAmMorrow? :trollface:

File: src/locales/en/common.json (hunk 1 above)
Comment (dasilvarosa) [reply src/locales/en/common.json]: Agree with @juan-corted:

"an ERC20 Token" 
"an Ethereum account"

**Comment by IAmMorrow** [comment]
@juan-cortes @gre 

Nice PR Bro

**Pull request merged**